Changelog.md

```diff
@@ -1,6 +1,28 @@
 ## libsdformat 13.X
 
-### libsdformat 13.3.0 (2022-12-12)
+### libsdformat 13.3.0 (2023-02-07)
+
+1. Add airspeed sensor
+    * [Pull request #1215](https://github.com/gazebosim/sdformat/pull/1215)
+
+1. Use `File.exist?` for Ruby 3.2 compatibility
+    * [Pull request #1216](https://github.com/gazebosim/sdformat/pull/1216)
+
+1. Make ThrowOrPrintError a free internal function
+    * [Pull request #1221](https://github.com/gazebosim/sdformat/pull/1221)
+    * [Pull request #1220](https://github.com/gazebosim/sdformat/pull/1220)
+
+1. macos workflow: don't upgrade existing packages
+    * [Pull request #1217](https://github.com/gazebosim/sdformat/pull/1217)
+
+1. update Param calls to use error vectors parameters
+    * [Pull request #1140](https://github.com/gazebosim/sdformat/pull/1140)
+
+1. ign -> gz Migrate Ignition Headers : sdformat
+    * [Pull request #1118](https://github.com/gazebosim/sdformat/pull/1118)
+
+1. Don't assume CMAKE_INSTALL_*DIR variables are relative
+    * [Pull request #1190](https://github.com/gazebosim/sdformat/pull/1190)
```

Comment:
we can remove this one too because it only applied to `sdf9` and couldn't be merged forward

Reply:
https://github.com/gazebosim/sdformat/pull/1229/commits/19cd95ebf75fea7db7ffe22444fffa4eb91a54fb